### ISSUE TITLE
Building on RPI3 & Tegra TK1 (ARM) Linux Ubuntu fails

### ISSUE THREAD

**Issue by sbfreddie** [seandepagnier/rtlsdr_pi#11]
oops!

**Comment by sbfreddie** [comment]
While attempting to build rtlsdr_pi on ARM I get this error:
Building the rtlsdr_pi plugin.....
Checking for Pre-Existing Plugin Rtlsdr_pi Folder.....
Checking to see if you have the latest updates.....
From https://github.com/seandepagnier/rtlsdr_pi
   1b9eeaa..84a4a8c  master     -> origin/master
Updating 1b9eeaa..84a4a8c
Fast-forward
 CMakeLists.txt | 7 +++++++
 1 file changed, 7 insertions(+)
Checking for rtlsdr_pi build directory.....
Removing CMakeCache.txt from build directory for a fresh build.....
Configuring rtlsdr_pi with Cmake...
-- *** Building rtlsdr_pi ***
-- *** Staging to build rtlsdr_pi ***
-- Found wxWidgets: TRUE  
-- Looking for XOpenDisplay in /usr/lib/arm-linux-gnueabihf/libX11.so;/usr/lib/arm-linux-gnueabihf/libXext.so
-- Looking for XOpenDisplay in /usr/lib/arm-linux-gnueabihf/libX11.so;/usr/lib/arm-linux-gnueabihf/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Found X11: /usr/lib/arm-linux-gnueabihf/libX11.so
-- Found OpenGL: /usr/lib/arm-linux-gnueabihf/libGL.so  
-- Found OpenGL...
--     Lib: /usr/lib/arm-linux-gnueabihf/libGLU.so/usr/lib/arm-linux-gnueabihf/libGL.so/usr/lib/arm-linux-gnueabihf/libSM.so/usr/lib/arm-linux-gnueabihf/libICE.so/usr/lib/arm-linux-gnueabihf/libX11.so/usr/lib/arm-linux-gnueabihf/libXext.so
--     Include: /usr/include
-- Found Gettext: /usr/bin/msgmerge (found version "0.19.3") 
-- Found BZip2: /usr/lib/arm-linux-gnueabihf/libbz2.so (found version "1.0.6") 
-- Looking for BZ2_bzCompressInit in /usr/lib/arm-linux-gnueabihf/libbz2.so
-- Looking for BZ2_bzCompressInit in /usr/lib/arm-linux-gnueabihf/libbz2.so - found
-- Found ZLIB: /usr/lib/arm-linux-gnueabihf/libz.so (found version "1.2.8") 
-- Configuring done
CMake Error at CMakeLists.txt:59 (ADD_LIBRARY):
  Cannot find source file:

    src/rtl-sdr-misc/aisdecoder/lib/protodec.c

  Tried extensions .c .C .c++ .cc .cpp .cxx .m .M .mm .h .hh .h++ .hm .hpp
  .hxx .in .txx


-- Build files have been written to: /home/pi/GitOpenCPN/Seans_Plugins/rtlsdr_pi/build
Installing rtlsdr_pi Failed

Cmake is looking for a folder that does not exist in the Git repository.

Thanks,
Ed

**Comment by svkatielee** [comment]
I also am having difficulty. I got it to compile and install but not tested yet by:
Cannot find source file:
    src/rtl-sdr-misc/aisdecoder/lib/protodec.c
from the build directory:
cp -r ../../aisdecoder/src/lib/ ../src/rtl-sdr-misc/aisdecoder/
then
cp ../../aisdecoder/config.h ../src/rtl-sdr-misc/aisdecoder/lib/

Of course this is not a good fix because config.h was created by cmake when I compiled aisdecoder in the previous steps. But it did make it finish the compile.
